frontend/src/pages/HomePage/HomePage.tsx

```diff
@@ -19,7 +20,13 @@ const HomePage = () => {
             <div className="home-page-content">
                 <div className="feed-section">
                     <h3>Uusimmat geokätköt</h3>
-                    <GeocacheList geocaches={geocaches} />
+                    {geocaches.map(cache =>
+                        <div key={cache.referenceCode} className="feed-link">
+                            <Link to={"/geocaches/" + cache.referenceCode} className="feed-link">
```

Comment:
Do you really suggest not to use original component for GeocacheList?

Reply:
What I'm suggesting is that the GeocacheList component should be modified, so that it contains links. The code that I posted was an example of what GeocacheList.tsx should return.

Reply:
I thought I am not allowed to modify component code

Reply:
Everything is allowed to be modified, if it makes sense to do so. Maybe the ticket description was a bit confusing. In any case, the idea was that GeocacheList should be modified.

Reply:
Okay. It's clear now

Reply:
I did the changes, but the GeocacheList test got now broken. It cannot find that screen text "Geo 1" inside the link, I guess. I'm trying to fix it next